src/firstboot/firstboot.c

```diff
@@ -540,14 +546,18 @@ static int prompt_timezone(int rfd) {
                 return 0;
         }
 
+        r = get_timezone(&zone);
+        if (r < 0)
+                return log_error_errno(r, "Cannot query timezone: %m");
```

Comment:
I think it is better to fall back to "UTC".

Reply:
I changed it to `UTC` for now.

I think it probably would make more sense to default these values to the current values. Usually you'd likely change the keymap and locale on an install medium before invoking something like `sd-firstboot` what you are about to install to.

Other than the current timezone I didn't see any function to get the current locale and keymap.